commands.json

```diff
@@ -17,6 +17,12 @@
             "cmds": [
                 "golemsp manifest-bundle add $_resources_dir"
             ]
+        },
+        {
+            "versions": ["v0.13.0-rc2"],
```

Comment:
IIUC 0.13.0-rc2 will be the first version with patched `ya-provider`

Reply:
It will be a new `default`, so maybe it should have a `"default": true`, otherwise new builds will not use it.
There is already version `v0.12` so it will not break current release.

Reply:
I believe it should be a default instead of `v0.12` (sorry, I must had deleted previous comment).